src/js/stores/AppsStore.js

```diff
@@ -134,6 +134,7 @@ function processApp(app) {
   app.healthWeight = getAppHealthWeight(app.health);
 
   app.tasks = lazy(app.tasks).map(function (task) {
+    task.id =  task.id || task.taskId;
```

Comment:
Actually not. Thats way beyond my marathon knowledge, but as this test was introduced in the mesos url component for some reason I'd rather keep it than removing it.